### PR TITLE
zh-translation:update Istio support-releases

### PR DESCRIPTION
**Pull request by hzde0128**
- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

update istio support releases

### CODE REVIEW

File: content/zh/docs/releases/supported-releases/index.md [hunk 1]
```diff
@@ -39,23 +39,29 @@ icon: cadence
...
+| 1.11       | 是           | 2021年8月12日  | ～2022年2月（预期）  | 1.19,1.20,1.21,1.22    | 1.16,1.17,1.18                     |
+| 1.10       | 是           | 2021年5月18日  | ～2021年11月（预期） | 1.18,1.19,1.20,1.21    | 1.16,1.17,1.22                     |
+| 1.9        | 是           | 2021年2月9日   | 2021年8月24日        | 1.17, 1.18, 1.19, 1.20 | 1.15, 1.16                         |
+| 1.8        | 否           | 2020年11月10日 | 2021年5月12日        | 1.16, 1.17, 1.18, 1.19 | 1.15                               |
+| 1.7        | 否           | 2020年8月21日  | 2021年2月25日        | 1.16, 1.17, 1.18       | 1.15                               |
+| 1.6 及更早 | 否           |                |                      |                        |                                    |
+
+{{< warning >}}
+Kubernetes 1.22移除了部分已弃用的API，因此，1.10.0之前的Istio版本将不再有效。如果要升级Kubernetes版本，请确保Istio的版本仍然支持。
+{{< /warning >}}
```
Comment (Xunzhuo) [inline content/zh/docs/releases/supported-releases/index.md:54]: 参考 English version，这里有个link

File: content/zh/docs/releases/supported-releases/index.md [hunk 2]
```diff
@@ -39,23 +39,29 @@ icon: cadence
...
 
 ## 没有通用漏洞披露（CVEs）的版本{#releases-without-known-Common-Vulnerabilities-and-Exposures}
```
Comment (Xunzhuo) [inline content/zh/docs/releases/supported-releases/index.md:56]: 没有已知的常见漏洞（CVEs）的版本

**Review by Xunzhuo — COMMENTED** [review]
minor comments, all other looks good to me

File: content/zh/docs/releases/supported-releases/index.md (hunk 1 above)
Comment (Xunzhuo) [reply content/zh/docs/releases/supported-releases/index.md:54]: https://kubernetes.io/blog/2021/07/14/upcoming-changes-in-kubernetes-1-22/

Comment (hzde0128) [reply content/zh/docs/releases/supported-releases/index.md:54]: tks

**Pull request closed without merging**